integration/docker/docker.go

```diff
@@ -0,0 +1,112 @@
+package docker
```

Comment:
Where are the functions in this file used besides in a test? Maybe I missed something here.

Reply:
This package is completely for testing. I copied this file from https://github.com/aquasecurity/harbor-scanner-trivy/blob/master/test/component/docker/docker.go. I separated the package so that it can be used from some tests.